### DISCUSSION TITLE
Progress indicator issue

### DISCUSSION THREAD

**Discussion by Minhazul-Islam18** [livewire/livewire#7378] [7. Bugs (archived)]
### Livewire version

3.0

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/Minhazul-Islam18/eSchoolManagementSystem

### Steps To Reproduce

    <div
        x-data="{ uploading: false, progress: 0 }"
        x-on:livewire-upload-start="uploading = true"
        x-on:livewire-upload-finish="uploading = false"
        x-on:livewire-upload-error="uploading = false"
        x-on:livewire-upload-progress="progress = $event.detail.progress"
    >
        
        <input type="file" wire:model="photo">
 
        
        <div x-show="uploading">
            <progress max="100" x-bind:value="progress"></progress>
        </div>
    </div>

I just use Livewire provided code, but it directly shows 100%, not 50/70/90. Either small or big files.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by jaskra** [comment]
I came here to report the same issue with Laravel 10.33, Livewire 3.2.1, and PHP 8.2.x. The livewire-upload-progress event fires once during a whole upload and it is at the end when it sets the value to 100. I also tried querying this in the progressCallback of uploadMultiple, but same thing, it only fires once at the end.

**Reply by timetorock** [reply]
Same for me, instantly shows 100

**Reply by jaskra** [reply]
This is odd but after writing that comment earlier today, I was changing up some different stuff in my uploader and the progress bar starts randomly working. Not entirely sure what I did to get it working as my code was pretty similar to the generic reference code in the first post of this thread. 

I did remove the x-on:livewire-upload-progress="progress = $event.detail.progress" in the parent div and am using the progressCallback in uploadMultiple to do essentially the same thing.

**Comment by thehands79** [comment]
[from this page](https://sinnbeck.dev/posts/making-a-complete-file-uploader-with-progressbar-using-livewire-and-alpinejs):
"Be aware that you might not be able to see the progress bar move at all, as the uploads are so fast when working locally. Therefor it might be a good idea to enable throttling in the browser, so you can actually see it progress: https://www.browserstack.com/guide/how-to-perform-network-throttling-in-chrome"

When I enabled network throttling I could see progress 1..2..3...etc.

**Comment by ramabie** [comment]
I also experienced the same problem
